control_panel_api/tests/__init__.py

```diff
@@ -1,5 +1,8 @@
 from django.conf import settings
 
+import control_panel_api.tests.filters
```

Comment:
If this import works then disregard what I said - I presumed `... import AppTestCase` etc would be required.